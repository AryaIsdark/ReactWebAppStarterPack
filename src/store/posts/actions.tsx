import {ActionTypes} from './types'
import mockData from './data.json'


export const loadData = () => {
    return {
        type: ActionTypes.SET_DATA,
        payload: {data: mockData}
    }
}