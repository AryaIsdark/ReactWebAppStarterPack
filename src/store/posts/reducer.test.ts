import reducer from "./reducer"
import * as actions from './actions'
export {reducer} from './reducer'

describe('store/posts/reducer',()=>{
    it('can load data', ()=>{
        expect(reducer(undefined, actions.loadData())).toMatchSnapshot();
    })
})