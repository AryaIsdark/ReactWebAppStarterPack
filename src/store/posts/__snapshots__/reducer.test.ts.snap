// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store/posts/reducer can load data 1`] = `
Object {
  "data": Array [],
  "hasError": false,
  "isLoading": false,
}
`;
